workflows/prognostic_c48_run/runtime/transformers/fv3fit.py

```diff
@@ -129,7 +128,7 @@ def non_negative_sphum_limiter(self, tendencies, inputs):
         return limited_tendencies
 
 
-TOTAL_PRECIP_RATE = "total_precipitation_rate"
+PRECIP_RATE = "surface_precipitation_rate"
```

Comment:
Is this a breaking change for the diagnostics/logged metrics? Not sure why the names needed to change.

Reply:
Change no longer in PR.